src/components/nimbleFileUploader/NimbleFileUploader.tsx

```diff
@@ -80,14 +82,14 @@ export const NimbleFileUploader: React.FC<NimbleFileUploaderProps> = ({
   const handleChange = async (event: any) => {
     const selectedFile = event.target.files[0];
     if (selectedFile) {
-      console.log(selectedFile);
       const fileName = selectedFile.name;
       const fileSize = Math.round(selectedFile.size / 1024);
       // const extension = fileName.substring(fileName.lastIndexOf('.') + 1);
 
       const data = await toBase64(selectedFile);
       onFileSelect(fileName, data, fileSize + 'KB', fileIdForEdit);
     }
```

Comment:
If the value is not cleared, the `onChange` event will not trigger if the user tries to upload the same file again.